translations/messages.fr.xlf

```diff
@@ -0,0 +1,3059 @@
+      <trans-unit id="5675185658977082941" datatype="html">
+        <source>Player <x id="PH" equiv-text="gameStatus.winner.value + 1"/></source>
+        <target>Joueur <x id="PH" equiv-text="gameStatus.winner.value + 1"/></target>
+      </trans-unit>
+      <trans-unit id="515727987845404926" datatype="html">
+        <source> It is your turn. </source>
+        <target> Ce n&apos;est pas votre tour !</target>
+      </trans-unit>
+      <trans-unit id="1992339978981826120" datatype="html">
+        <source> It is <x id="INTERPOLATION" equiv-text="{{ currentPlayer }}"/>&apos;s turn. </source>
+        <target> C&apos;est au tour de <x id="INTERPOLATION" equiv-text="{{ currentPlayer }}"/></target>
+      </trans-unit>
+      <trans-unit id="5217544685559946950" datatype="html">
+        <source> Resign </source>
+        <target> Abandonner </target>
+      </trans-unit>
+      <trans-unit id="4084373978972803762" datatype="html">
+        <source> Propose a draw </source>
+        <target> Proposer un match nul </target>
+      </trans-unit>
+      <trans-unit id="772509734777361990" datatype="html">
+        <source> Accept draw </source>
+        <target> Accepter un match nul </target>
+      </trans-unit>
+      <trans-unit id="2984356495884926241" datatype="html">
+        <source> Reject draw </source>
+        <target> Refuser un match nul </target>
+      </trans-unit>
+      <trans-unit id="101333213054798052" datatype="html">
+        <source> Accept take back </source>
+        <target> Autoriser à reprendre un tour </target>
+      </trans-unit>
+      <trans-unit id="719941689314037005" datatype="html">
+        <source> Reject take back </source>
+        <target> Refuser de reprendre un tour </target>
+      </trans-unit>
+      <trans-unit id="1601597703777069856" datatype="html">
+        <source><x id="INTERPOLATION" equiv-text="{{ currentPart.getLoser() }}"/> has reached their time limit. You won.</source>
+        <target><x id="INTERPOLATION" equiv-text="{{ currentPart.getLoser() }}"/> a épuisé son temps. Vous avez gagné.</target>
+      </trans-unit>
+      <trans-unit id="7814033294193818165" datatype="html">
+        <source>You reached your time limit.</source>
+        <target>Vous avez dépassé la limite de temps.</target>
+      </trans-unit>
+      <trans-unit id="2658606681190400847" datatype="html">
+        <source> Ask to take back one move </source>
+        <target> Demander à reprendre un coup </target>
+      </trans-unit>
+      <trans-unit id="2826140657122926749" datatype="html">
+        <source>You resigned.</source>
+        <target>Vous avez abandonné.</target>
+      </trans-unit>
+      <trans-unit id="2324913504104154958" datatype="html">
+        <source><x id="INTERPOLATION" equiv-text="{{ currentPart.getLoser() }}"/> has reached their time limit.</source>
+        <target><x id="INTERPOLATION" equiv-text="{{ currentPart.getLoser() }}"/> a épuisé son temps.</target>
+      </trans-unit>
```

Comment:
"run out of time" semble signifier "manque de temps".
Mais "has no more time" serait mieux ou pas ?

Reply:
has no more time je dirais non. Plutôt "XXX ran out of time"?

Reply:
Je crois que "ran out of time" c'est plus "manquait de temps", mais again, voyons peut être ce qu'ils disent sur BGA (Board Game Arena, sur lesquelles la présidente de cercle du CEI m'as avoué s'être inscrit alors qu'elle dit "je viens tester MGP à l'occasion!" à chaque fois qu'on se voit LEL :D !

Reply:
Tiens pour le coup, on séparerait pas le "you won" du reste pour n'avoir qu'un seul "reach out of time" dans la liste ?